general/package/baresip-openipc/Config.in

```diff
@@ -1,12 +1,13 @@
 config BR2_PACKAGE_BARESIP_OPENIPC
-	bool "baresip-openipc"
-	select BR2_PACKAGE_LIBRE_OPENIPC
-	select BR2_PACKAGE_LIBREM_OPENIPC
-	default n
-	help
-	  Baresip is a portable and modular SIP User-Agent with audio 
-	  and video support. Copyright (c) 2010 - 2022 Alfred E. Heggestad 
-	  and Contributors Distributed under BSD license.
-
-	  https://github.com/baresip/baresip
+    bool "baresip-openipc"
+    default n
+    depends on BR2_PACKAGE_LIBRE_OPENIPC
+    depends on BR2_PACKAGE_LIBREM_OPENIPC
```

Comment:
Having both select and depends on creates circular dependency. You should only use one or another.